### PR TITLE
sql: add doc for bit functions and operators

### PR DESCRIPTION
**Pull request by XuHuaiyu**
PTAL @andelf

### CODE REVIEW

File: sql/bit-functions-and-operators.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: TiDB 用户文档
+category: user guide
+---
+
+## 位函数和操作符
```
Comment (andelf) [inline sql/bit-functions-and-operators.md]: better level 1

File: sql/bit-functions-and-operators.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+TiDB 中位函数和操作符的使用方法与 MySQL 基本一致，详情参见: [Bit Functions and Operators](https://dev.mysql.com/doc/refman/5.7/en/bit-functions.html).
+
+**位函数和操作符表**
+
+| 函数和操作符名                                  | 功能描述            |
+| ---------------------------------------- | --------------- |
```
Comment (andelf) [inline sql/bit-functions-and-operators.md]: wrong column width indicator.
shorter first, longer second.

**Comment by XuHuaiyu** [comment]
PTAL @andelf

File: sql/bit-functions-and-operators.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+**位函数和操作符表**
+
+| 函数和操作符名 | 功能描述                              |
+| -------------- | ------------------------------------- |
+| [`BIT_COUNT()`](https://dev.mysql.com/doc/refman/5.7/en/bit-functions.html#function_bit-count) | 返回参数二进制表示中为1的位数 |
+| [`&`](https://dev.mysql.com/doc/refman/5.7/en/bit-functions.html#operator_bitwise-and) | 位与              |
+| [`~`](https://dev.mysql.com/doc/refman/5.7/en/bit-functions.html#operator_bitwise-invert) | 按位取反            |
+| [`|`](https://dev.mysql.com/doc/refman/5.7/en/bit-functions.html#operator_bitwise-or) | 位或              |
+| [`^`](https://dev.mysql.com/doc/refman/5.7/en/bit-functions.html#operator_bitwise-xor) | 位亦或             |
```
Comment (andelf) [inline sql/bit-functions-and-operators.md]: typo

File: sql/bit-functions-and-operators.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+# 位函数和操作符
+
+TiDB 中位函数和操作符的使用方法与 MySQL 基本一致，详情参见: [Bit Functions and Operators](https://dev.mysql.com/doc/refman/5.7/en/bit-functions.html).
+
+**位函数和操作符表**
+
+| 函数和操作符名 | 功能描述                              |
+| -------------- | ------------------------------------- |
+| [`BIT_COUNT()`](https://dev.mysql.com/doc/refman/5.7/en/bit-functions.html#function_bit-count) | 返回参数二进制表示中为1的位数 |
```
Comment (andelf) [inline sql/bit-functions-and-operators.md]: 位数  -> 个数

**Pull request merged**